test/test_jit.py

```diff
@@ -10221,6 +10221,12 @@ def cdist_fn():
 
         self.checkScript(cdist_fn, ())
 
+        def norm():
+            c = torch.tensor([[1, 2, 3], [-1, 1, 4]], dtype=torch.float)
+            return torch.norm(c, p="fro"), torch.norm(c, p="nuc"), torch.norm(c), torch.norm(c, p=.5)
```

Comment:
Is it not necessary to test with different function arguments, or are they left out since they are covered by standard tests in test_torch

Reply:
Yea, I was mostly testing that it compiles with different types of arguments. Since test_torch tests with different argument values and the code is the same for python/TorchScript, that should be sufficient